### ISSUE TITLE
Add support for mounting by tag

### ISSUE THREAD

**Issue by alvarlagerlof** [coreos/ignition#1281]
# Feature Request #

## Environment ##

Local using virsh and qemu

## Desired Feature ##

I would like to be able to mount things like virtio-fs that uses tags to mount in the ignition file directly, instead of having to create systemd units that run the `mount -t` command and make everything more complex.

**Comment by bgilbert** [comment]
If you want the provisioned OS to automatically mount a filesystem, we don't recommend writing `mount` commands inside a systemd service unit; you should be able to write a [systemd mount unit](https://www.freedesktop.org/software/systemd/man/systemd.mount.html) instead.  Does that approach work for you?

It might help if you could post a minimal Ignition config showing the workaround you're using now.

**Comment by alvarlagerlof** [comment]
> you should be able to write a systemd mount unit instead

No, I haven't been able to make that work. I didn't find anything in /dev to mount the `What=` to that corresponds to the tag and doesn't change.
